Libraries/Components/Button.js

```diff
@@ -14,7 +14,6 @@
 const ColorPropType = require('ColorPropType');
 const Platform = require('Platform');
 const React = require('React');
-const PropTypes = require('prop-types');
```

Comment:
Can we make this change in a separate PR?

Reply:
That means, Remaining(const PropTypes = require...) is better?